e2e-tests/cypress/e2e/journeys/insurance/your-business/broker/broker-page.spec.js

```diff
@@ -156,16 +160,60 @@ context('Insurance - Your business - Broker Page - As an Exporter I want to conf
     cy.checkText(saveAndBackButton(), BUTTONS.SAVE_AND_BACK);
   });
 
-  describe('when submitting an empty form', () => {
-    const errorMessage = ERROR_MESSAGE_BROKER.IS_EMPTY;
+  describe('form submission', () => {
+    describe('when submitting an empty form', () => {
+      const errorMessage = ERROR_MESSAGE_BROKER.IS_EMPTY;
 
-    it(`should display validation errors if ${USING_BROKER} radio not selected`, () => {
+      it(`should display validation errors if ${USING_BROKER} radio not selected`, () => {
       // visit url to refresh form and radios
-      cy.navigateToUrl(url);
+        cy.navigateToUrl(url);
 
-      const { field, numberOfExpectedErrors, errorIndex } = ERROR_ASSERTIONS;
+        const { field, numberOfExpectedErrors, errorIndex } = ERROR_ASSERTIONS;
 
-      cy.submitAndAssertRadioErrors(field, errorIndex, numberOfExpectedErrors, errorMessage);
+        cy.submitAndAssertRadioErrors(field, errorIndex, numberOfExpectedErrors, errorMessage);
+      });
+    });
+
+    describe('when submitting a fully filled form', () => {
+      describe(`when selecting yes for ${USING_BROKER}`, () => {
+        it(`should redirect to ${CHECK_YOUR_ANSWERS} page`, () => {
+          cy.completeAndSubmitBrokerForm();
+
+          cy.url().should('eq', checkYourAnswersUrl);
+        });
+
+        describe('when going back to the page', () => {
+          it('should have the submitted values', () => {
+            cy.navigateToUrl(url);
+
+            broker[USING_BROKER].yesRadioInput().should('be.checked');
+            cy.checkValue(broker[NAME].input(), application.EXPORTER_BROKER[NAME]);
+            cy.checkValue(broker[ADDRESS_LINE_1].input(), application.EXPORTER_BROKER[ADDRESS_LINE_1]);
+            cy.checkValue(broker[ADDRESS_LINE_2].input(), '');
+            cy.checkValue(broker[TOWN].input(), application.EXPORTER_BROKER[TOWN]);
+            cy.checkValue(broker[COUNTY].input(), '');
```

Comment:
Is there reason why address line 2 and country are not being provided/tested here? Would be great to have full data set as much as possible